test/framework/options.py

```diff
@@ -5857,6 +5857,41 @@ def test_sanity_check_only(self):
             regex = re.compile(error_pattern)
             self.assertTrue(regex.search(error_msg), "Pattern '%s' should be found in: %s" % (regex.pattern, error_msg))
 
+        # failing sanity check for extension can be bypassed via --skip-extensions
+        self.eb_main(args + ['--skip-extensions'], do_build=True, return_error=True)
```

Comment:
There's no test here is there, so what's the point?

Reply:
Yeah, what's the use of this?

Reply:
If the sanity check fails, then an error is raised, so this would cause the test to fail.

I can add another quick check for the module file after this though, to make that a bit more clear.